Feature/configure grafana

dougszumski opened the pull request:
Monasca isn't configured on the alt-1 controller, but this is useful for testing against Grafana which is deployed on it.

dougszumski committed:
Configure Grafana for Monasca

dougszumski committed:
Revert "Rename the local Grafana admin user"

This is now set in Kayobe by default.

This reverts commit 19a107679f4db7a6c958a60b33bee34e21fc8383.

markgoddard commented on etc/kayobe/grafana.yml:
```diff
@@ -0,0 +1,58 @@
+
+# Grafana local admin user name. If you are deploying Monasca Grafana this
+# should not conflict with an OpenStack user name.
+#grafana_local_admin_user_name:
```
If you go with my suggestion of using admin as the default in kayobe, this will need to be overridden.

dougszumski replied:
done

markgoddard commented on etc/kayobe/grafana.yml:
```diff
@@ -0,0 +1,58 @@
+# The path, relative to the grafana_monitoring_node_dashboard_repo_path
+# containing the dashboards. Eg. /prometheus/control_plane
+#grafana_monitoring_node_dashboard_path:
+grafana_monitoring_node_dashboard_path: "/monasca/control_plane"
```
As in kayobe review, could move this with the other variables?

dougszumski replied:
done

dougszumski committed:
Update Grafana configuration following review comments

dougszumski committed:
Specify the repo branch to use

dougszumski committed:
Use StackHPC fork of Monasca Grafana Plugin to support log API

We should revert back to the official plugin once our changes are
merged upstream.

For some reason piping curl output directly to tar didn't work in
the container so a tmp file was used.
